packages.yml

```diff
@@ -1,3 +1,7 @@
 packages:
- - package: fivetran/netsuite_source
-   version: [">=0.8.0", "<0.9.0"]
+#  - package: fivetran/netsuite_source
+#    version: [">=0.9.0", "<0.10.0"]
+
+- git: https://github.com/fivetran/dbt_netsuite_source.git
+  revision: feature/subsidiary-currency
+  warn-unpinned: false
```

Comment:
Obligatory reminder to update before merge

Reply:
Will keep as branch during pre-release.